src/routes/jira/events/jira-app-creation-router.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+import { JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
+import { JiraAppCreationPost } from "../server/jira-app-creation-post";
+import { JiraAppCreationGet } from "../server/jira-app-creation-get";
+import { csrfMiddleware } from "middleware/csrf-middleware";
+
+export const JiraAppCreationRouter = Router();
+
+JiraAppCreationRouter.route("/")
+	.post(JiraAppCreationPost)
```

Comment:
Pretty sure the post needs JWT auth here in the body with csrf :)

Reply:
I was worried you guys would say that. It fails with: `ForbiddenError: invalid csrf token`

Reply:
I updated the post to pass the csrf in (same thing I had done locally before opening this PR). I checked the csrf is being passed in but it fails with the above :blobconfused: :halp: